libs/kano-networking/includes/ifaces.h

```diff
@@ -0,0 +1,20 @@
+/**
+ * ifaces.h
+ *
+ * Copyright (C) 2016 Kano Computing Ltd.
+ * License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * Methods for interracting with interfaces
+ */
+
+#ifndef __KANO_NETWORKING_IFACES__
+#define __KANO_NETWORKING_IFACES__
+
+enum ERRORS {
+    E_NO_INTERFACES = -1
+};
+
+int select_iface(char * iface_type, char **iface_name);
+int check_iface_type(char * iface_type);
```

Comment:
Personally I like my typedefs, so I would typedef the enum and then have that be the return value of these two fns, but it is clear as it is anyway